fabric-common/lib/Channel.js

```diff
@@ -11,7 +11,7 @@ const DiscoveryService = require('./DiscoveryService.js');
 const Endorsement = require('./Endorsement.js');
 const Commit = require('./Commit.js');
 const Query = require('./Query.js');
-const fabprotos = require('fabric-protos');
+const fabproto6 = require('fabric-protos');
```

Comment:
What is the special reason for the '6' suffix?

Reply:
the conversion is based on protobufjs v6